tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+<?php
+/**
+* Plugin Name: Eventbrite Tickets Extension: Additional Options
+* Description: Adds a new Eventbrite options section to the bottom of wp-admin > Events > Settings > Imports tab. Options include text above or below iframe, iframe height, moving ticket location on Single Event view, displaying tickets for Private Eventbrite events, change API URL (e.g. from .com to .co.uk), and more.
+* Version: 1.0
+* Extension Class: Tribe__Extension__Eventbrite_Addl_Opts
+* Author: Modern Tribe, Inc.
+* Author URI: http://m.tri.be/1971
+* License: GPLv2 or later
+* License URI: https://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+* Extension main class, class begins loading on init() function.
+*/
+class Tribe__Extension__Eventbrite_Addl_Opts extends Tribe__Extension {
+
+	protected $opts_prefix = 'tribe_ext_eventbrite_opts_';
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 *
+	 * @access protected
+	 */
+	protected function construct() {
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Events__Main', '4.4' ); // due to using tribe() from Tribe-Common as of 4.4
```

Comment:
@elimn, I initially thought you meant line 38:
`// Each plugin required by this extension`

but, re-reading this, I think you're referring to the comments at the end of lines 39-40.

Or maybe you are referring to all 3 comments. I got rid of the comments after lines 39 and 40 but not 38.